doc/specifications/rest/REST-API-V2.rst

```diff
@@ -9,6 +9,22 @@ eZ Platform REST API V2
 General considerations
 ======================
 
+Specifying a siteaccess
+-----------
+
+In order to specify a siteaccess when talking to the REST API, a custom header, "X-Siteaccess", needs to be provided.
```

Comment:
```suggestion
In order to specify a SiteAccess when talking to the REST API, a custom `X-Siteaccess` header has to be provided.
```

Reply:
@mateuszbieniek seems you need to apply this :)

@michalmilc90 Does this mean this is approved from Doc team as well once this is in? ;)

Reply:
@andrerom yes! 😉